src/scripts/homebuttonpage.js

```diff
@@ -0,0 +1,17 @@
+function homebuttonpage() {
+  const content = document.querySelector('.content');
+  const homeButton = document.querySelector('.home');
+  homeButton.addEventListener('click', () => {
+    const contactContent = document.querySelector('.contact_content');
```

Comment:
- [ ] This isn't the right way to export a module. Do this instead: `export default homebuttonpage;`. **Fix all other exported modules also.**